### ISSUE TITLE
Cannot import bookings

### ISSUE THREAD

**Issue by Structed** [lg2de/SimpleAccounting#138]
# Description
I'd like to start by importing bookings. However, I can not
* Select an account to import into (probably because I do not have an import config)
* I can't seem to find a way to create an import config for an account
* I cannot select a file/location to import from

## Steps to reproduce
1. Open SimpleAccounting 
2. Select `Project` --> `New` to create a new project
3. Click `Actions` --> `Import Bookings`
4. No way to select an account
5. No way to select where to import from

## Expected behaviour
1. Open SimpleAccounting 
2. Select `Project` --> `New` to create a new project
3. Click `Actions` --> `Import Bookings`
4. Select an account from teh dropdown
5. Select a file to import from
6. Click `Book partially` or `Bool all` to import

## Screenshots
![image](https://user-images.githubusercontent.com/29922/113581017-d3b96b80-9626-11eb-8723-f80c4d381268.png)

## Environment
 - OS: Windows 10 Pro 20H2 Vuild 19042.867
 - SimpleAccounting Version 2.1.0
- .NET SDKs installed:
  - 5.0.103 [C:\Program Files\dotnet\sdk]
  - 5.0.201 [C:\Program Files\dotnet\sdk]
- .NET runtimes installed:
  - Microsoft.AspNetCore.All 2.1.26 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  - Microsoft.AspNetCore.App 2.1.26 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  - Microsoft.AspNetCore.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  - Microsoft.AspNetCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  - Microsoft.AspNetCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  - Microsoft.NETCore.App 2.1.26 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  - Microsoft.NETCore.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  - Microsoft.NETCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  - Microsoft.NETCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  - Microsoft.WindowsDesktop.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  - Microsoft.WindowsDesktop.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  - Microsoft.WindowsDesktop.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

**Comment by lg2de** [comment]
Currently there is no visual configuration option to activate import mapping for an account.
You have to write it manually into the file. It is an XML file.
There is also an XSD describing the structure: https://lg2de.github.io/SimpleAccounting/AccountingData.xsd

Here is a (german) example for the account configuration:
```xml
<Account ID="1100" Name="Vereinskonto" Type="Asset">
  <ImportMapping>
    <Columns>
      <Column Target="Date" Source="Buchungstag" />
      <Column Target="Name" Source="Zahlungsempfänger" />
      <Column Target="Text" Source="Vorgang/Verwendungszweck" IgnorePattern="I ?B ?A ?N ?:? D ?E ?.*$" />
      <Column Target="Value" Source="Umsatz" />
    </Columns>
    <Patterns>
      <Pattern Expression="(?i)Jahresbeitrag" AccountID="4100" />
      <Pattern Expression="(?i)Mitgliedsbeitrag" AccountID="4100" />
      <Pattern Expression="(?i)Vereinsbeitrag" AccountID="4100" />
      <Pattern Expression="(?i)beitrag" Value="2000" AccountID="4100" />
      <Pattern Expression="(?i)spende elternbeitrag" AccountID="10001" />
      <Pattern Expression="(?i)spende" AccountID="4200" />
      <Pattern Expression="(?i)vereinsjub" AccountID="6203" />
    </Patterns>
  </ImportMapping>
</Account>
```
The section `Columns` is required to activate the import for this account. The account will be available in the selection list of the import dialog. This configuration defines the mapping between the columns in the CSV file (`Source`) and the internal database (`Target`).

The section `Patterns` is optional to pre-assign some entries regarding the remote account.

If you are a developer you are invited to contribute the visual configurator.
Otherwise I hope that the information above helps you to activate the import. Please ask for further assistance if required.

**Comment by lg2de** [comment]
I've added a simple configuration dialog. It will be available in version 2.2.
